File: tests/integration/deleted_with/python/fails_on_delete.py
# Copyright 2016-2023, KhulnaSoft Ltd.  All rights reserved.

from typing import Optional

import codeinfra

class FailsOnDelete(codeinfra.CustomResource):
    def __init__(self,
                 resource_name: str,
                 opts: Optional[codeinfra.ResourceOptions] = None):
        super().__init__("testprovider:index:FailsOnDelete", resource_name, {}, opts)

File: tests/integration/deleted_with/nodejs/failsOnDelete.ts
// Copyright 2016-2023, KhulnaSoft Ltd.  All rights reserved.

import * as codeinfra from "@codeinfra/codeinfra";

export class FailsOnDelete extends codeinfra.CustomResource {
    constructor(name: string, opts?: codeinfra.CustomResourceOptions) {
        super("testprovider:index:FailsOnDelete", name, {}, opts);
    }
}
